src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/BytesExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System.Text;
+using System;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions
+{
+    internal static class BytesExtensions
+    {
+        public static string ToBase64Url(this byte[] bytes)
```

Comment:
Can you put a summary that converts byte array to b64 URL. Trims trailing `=` characters and also link to b64 spec.

Reply:
The link to the spec: https://datatracker.ietf.org/doc/html/rfc4648#section-5